zerver/lib/actions.py

```diff
@@ -4382,11 +4382,15 @@ class MessageUpdateUserInfoResult(TypedDict):
     message_user_ids: Set[int]
     mention_user_ids: Set[int]
 
-def notify_topic_moved_streams(user_profile: UserProfile,
```

Comment:
Split this into two commits -> Rename of function and allowing using of custom msg strings. Maybe `send_message_moved_breadcrumbs` is a better name for this function since I expect we will use this function when moving both topics and streams in the future.